README.md

```diff
@@ -1 +1,17 @@
-# koach
\ No newline at end of file
+# koach
+
+Koach collects observability data from relay server and save it in database. Koach also provided gRPC method to query this data, which currently used by the KubeArmor CLI.
+
+## Quick Start
+
+To start koach service, build the program first using the `make build` command, then go to the koach directory and run this command
```

Comment:
Can you specify the prerequisites? If someone follows this guide, they wont get any logs. Koach connects to kubearmor to get the logs.